pysmt/cmd/installers/bdd.py

```diff
@@ -44,7 +44,7 @@ def compile(self):
             makefile = "Makefile_64bit"
 
         # Build the pycudd
-        command = ['python%s-config' % self.python_version, '--includes']
+        command = ['python-config', '--includes']
```

Comment:
This is not correct when multiple versions of python are installed (e.g., 3.5 and 3.6). python-config maps to the default interpreter (the one you would get when you call python), while python%s-config takes into account the version of python used by the interpreter calling the installer.

Reply:
Indeed, but since `python3.6-config` is not installed on travis I see 2 strategies:
1) Try to use `python{version}-config` and if it fails try with `python-config`
2) Avoid `python-config` and rely on the `sysconfig` module as it is done by the btor `configure.sh` script: 
```
  pyinc=`$PYTHON -c "import sysconfig; print(sysconfig.get_config_var('CONFINCLUDEPY'))"`
  pylib=`$PYTHON -c "import sysconfig; print(sysconfig.get_config_var('BINLIBDEST'))"`
```

Reply:
Is this a Travis issue or an Ubuntu issue?

I would go with solution 1. printing a warning message when falling back to python-config.

Solution 2. does not work within a virtual environment. When the virtual environment is constructed, the interpreter is copied in the virtual env. sysconfig uses the information from the interpreter and is not aware that it is in a virtual env. python-config lives within the virtualenv, and therefore can provide the correct path. 
You can check this by comparing the output of ``python-config --includes`` and ``python -m sysconfig | grep CONFINC`` from within a virtual environment.

Reply:
On my Ubuntu machine I do have the `python{version}-config`, so I guess it is either a Travis issue or a virtual-env issue...

Reply:
Let's go for solution 1 then